lms/templates/content_item_selection/new_content_item_selection.html.jinja2

```diff
@@ -230,8 +253,10 @@
     function pickerCallback(data) {
       var docCount = data.docs && data.docs.length;
       if (data.action == google.picker.Action.PICKED && docCount === 1) {
```

Comment:
Is it possible for more than 1 document to get picked? If so we should return an error to the user to let them know they should only pick 1 document at a time.

Reply:
The picker is configured to only allow a single selection